src/app/pages/OnDemandTvPage/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { shape, string, number } from 'prop-types';
+import MetadataContainer from '../../containers/Metadata';
+import { GelPageGrid } from '#app/components/Grid';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+const getGroups = (zero, one, two, three, four, five) => ({
```

Comment:
Out of scope for this,  but the Gel grid should probably just support this pattern better - seen this code a few times now

Something like `<GelGrid columns={[6, 6, 6, 6, 8, 20]}>`

Reply:
Agreed. I'll add a note to the board